app.rb

```diff
@@ -1,9 +1,23 @@
 require 'sinatra/base'
+require_relative './lib/message'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
```

Comment:
It's good to remove any extraneous code from your implementation before making a Pull Request - there'll be lots of testing and exploration as you work on something, but it tidies it up to remove anything that is no longer needed for the program to meet its specification.